extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -38,7 +39,9 @@
 
 @SuppressWarnings("unused")
 public class LambdaHttpHandler implements RequestHandler<AwsProxyRequest, AwsProxyResponse> {
-    private static final Logger log = Logger.getLogger("quarkus.amazon.lambda.http");
+    private static final Logger LOG = Logger.getLogger("quarkus.amazon.lambda.http");
```

Comment:
Please avoid doing subjective changes like this  (changing variable names) in the future.

Reply:
Sorry, this is standard constant naming style. I assume it didn't hurts to change them.

Reply:
Reverted